README.md

```diff
@@ -64,6 +64,8 @@ createClient({
 
 You can also use discrete parameters, UNIX sockets, and even TLS to connect. Details can be found in the [client configuration guide](./docs/client-configuration.md).
 
+To check if the the client is connected and ready to send commands, use `client.isReady` which returns a boolean. 
```

Comment:
can you please mention `isOpen` as well? 🙏

Reply:
I was trying to work out - what's a situation where you could get different values for `isOpen` and `isReady`... I thinnk understanding this would help me phrase something.

Reply:
const connectPromise = client.connect();
console.log(client.isOpen, client.isReady); // false, true
await connectPromise();
console.log(client.isOpen, client.isReady); // true, true

(same when reconnecting due to a network error)

Reply:
Done!